packages/unlock-js/src/index.ts

```diff
@@ -1,5 +1,5 @@
 export { default as Web3Service } from './web3Service'
 export { default as WalletService } from './walletService'
 export { getCurrentProvider, getWeb3Provider } from './providers'
-export const latestUnlock = 'v10'
-export const latestPublicLock = 'v8'
```

Comment:
Oops. Did we have the wrong value here?

Reply:
yes, seems it was the wrong value for public lock :/